3.1/imports/heading-row.md

```diff
@@ -111,6 +111,9 @@ Or you can then set this new formatter in a service provider.
 ```php
 HeadingRowFormatter::default('custom');
 ```
+:::warning
+`HeadingRowFormatter` should never be inside the import method. This causes unexpected behaviour when used with ChunkReading and queues.
```

Comment:
I would prefer having a message to suggest the right way of doing it instead of only warning. Could you update your PR ?